deploy/cr.yaml

```diff
@@ -25,3 +25,6 @@ spec:
       fluentd:
         nodeSelector:
           logging-infra-fluentd: "true"
+      rsyslog:
```

Comment:
I was actually wondering about scenarios like this... 
For this case, we would want someone to configure to install with either `rsyslog` **or** `fluentd`. So should we just provide two different CR examples?

Reply:
I suppose, although the `type:` is the thing which controls which one is used.